userspace/libsinsp/filterchecks.cpp

```diff
@@ -3537,6 +3680,12 @@ uint8_t* sinsp_filter_check_thread::extract(sinsp_evt *evt, OUT uint32_t* len, b
 	case TYPE_IS_EXE_FROM_MEMFD:
 		m_tbool = tinfo->m_exe_from_memfd;
 		RETURN_EXTRACT_VAR(m_tbool);
+	case TYPE_IS_SID_LEADER:
+		m_tbool = tinfo->m_sid == tinfo->m_vpid ? true : false;
```

Comment:
why do we use 
```bash
tinfo->m_sid == tinfo->m_vpid
```

instead of

```bash
tinfo->m_sid == tinfo->m_pid
```

`m_sid` is referred to the init namespace so probably we should use `m_pid` or am I missing something?

Reply:
actually here is correct to use `vpid` since `m_sid` could be also referred to a pid namespace :/